storageserver/src/tests/testhelper.cpp

```diff
@@ -66,7 +66,7 @@ vdstestlib::DirConfig getStandardConfig(bool storagenode) {
     // Easier to see what goes wrong with only 1 thread per disk.
     config->set("minimum_file_meta_slots", "2");
     config->set("minimum_file_header_block_size", "368");
-    config->set("minimum_file_size", "4096");
+    config->set("minimum_file_size", "4_Ki");
```

Comment:
What will be the actual string here? If "4_Ki", I guess we must change to "4096".